source/mongostat/mongostat-behavior.txt

```diff
@@ -39,6 +39,12 @@ with the :option:`--auth <mongod.--auth>` option, specify the
 <mongostat --password>` options, and the connecting user must have the
 :authaction:`serverStatus` privilege action on the cluster resources.
 
+.. important::
+
+   Starting in MongoDB 8.0, to use ``mongostat`` to connect to a
+   sharded cluster, the user must also have the
+   :authrole:`directShardOperations` role.
+
 The built-in role :authrole:`clusterMonitor` provides this privilege as
```

Comment:
The copy in the admonition looks good, but I think now we need to rework the following paragraph slightly. We're saying "The built-in role`clusterMonitor` provides this privilege", where "this privilege" previously referred to `serverStatus`, but now it's not clear whether "this privilege" refers to `serverStatus` or `directShardOperations`.

Reply:
@jeff-allen-mongo, Fixed.